### PR TITLE
Minor changes to DEV-INTRO.md

### PR DESCRIPTION
**Pull request by dlebauer**
relatively minor changes to DEV-INTRO

## Description

Notes on installing docker

## Motivation and Context

Just trying to be helpful

## Review Time Estimate

- [ ] Immediately
- [ ] Within one week
- [x] When possible
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) 
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My change requires a change to the documentation.
- [ ] I have updated the CHANGELOG.md.
- [x] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: DEV-INTRO.md [hunk 1]
```diff
@@ -55,7 +62,11 @@ The steps in this section only need to be done the fist time you start working w
 
 #### .env file
 
-You can copy the [`env.example`](docker/env.example) file as .env in your pecan folder. The variables we want to modify are:
+You can copy the [`docker/env.example`](docker/env.example) file as .env in your pecan folder. The variables we want to modify are:
+
+```sh
```
Comment (robkooper) [inline DEV-INTRO.md:67]: make sure to add the windows version as well

File: DEV-INTRO.md [hunk 2]
```diff
@@ -10,19 +10,26 @@ To clone the PEcAn repository:
...
+cd pecan
 # alternatively, if you haven't set up ssh keys with GitHub
 # git clone https://github.com/PecanProject/pecan
 ```
 
 ## Developing in Docker
 
+### Installing Docker 
+
+To install Docker and docker-compose, see the docker documentation for installing
```
Comment (robkooper) [inline DEV-INTRO.md:22]: maybe refer to the docker documentation in the manual as well.

File: DEV-INTRO.md (hunk 1 above)
Comment (dlebauer) [reply DEV-INTRO.md:71]: added

Comment (dlebauer) [reply DEV-INTRO.md:71]: done

File: DEV-INTRO.md (hunk 2 above)
Comment (dlebauer) [reply DEV-INTRO.md]: done

**Pull request merged**